Update README.md

vangewc opened the pull request:
docs: adding msg as test

vangewc committed:
Update README.md

<docs>:<adding msg as test>

vangewc commented:
I have a question about this:
![image](https://github.com/user-attachments/assets/119e89d0-57f7-420f-ad12-667faafea00c)
@htessaro

htessaro commented:
@vangewc , the information added in the GitHub desktop, is the commit message. You can make as many commits as necessary. It depends on the number of files you are changing, for example. For example, after finishing a tutorial you can create a commit. But, if you didn't finish it in one day, you can commit your changes at the end of the day, and continue tomorrow, and add a new commit with the changes you did in the next day in the same file. In sumary, the commits are used to register the changes you did to the project.

![image](https://github.com/user-attachments/assets/b276c5c2-0f93-4f87-bde6-e1b2142de1b6)

The information in the second image refers to the Pull Request. Here, you will identify and describe the Pull Request, which is composed by all commits you did to the branch you were working on. The Pull Request is used to notify the other users that you want to merge the changes you made to the project (using your branch) into the main branch. The Pull Request details are more general, while the commits messages are more specific about what you have changed.

However, since we are creating documentation, we normally will be changing one file. Thus, the Pull request information details will be similar to the commits, since the idea is to create one branch for each file you'll be creating/editing, resulting in one Pull request for each file, basically.

vangewc committed:
Feat: Add April 2025 internal release notes and assets

Release Notes for April 15 and 29.
April 15 has images, while April 29 doesn't have any yet.

vangewc committed:
MODS. to Release Notes

Requested modifications applied to RN April 15th
